packages/subgraph/schema.graphql

```diff
@@ -462,152 +462,356 @@ type StandardCallback @entity {
   magicNumber: Bytes!
 }
 
-# DAO
+#############################
+########### DAO #############
+#############################
 
+"""
+The Aragon DAOs deployed using the `DAOFactory` SC `createDAO` function and registered in the `DAORegistry` SC. From this entity you are able to query
+any relevant information related to your DAO.
+"""
 type Dao @entity {
-  id: ID! # use address as id
+  " The address of the deployed DAO contract registered in the `DAORegistry` SC. "
+  id: ID!
+
+  " ENS DAO subdomain, if added while creating the DAO. "
   subdomain: String
+
+  " The address of the creator who called (msg.sender) `createDao` in the `DAOFactory` SC. "
   creator: Bytes!
+
+  " The DAO uri used with [EIP-4824](https://eips.ethereum.org/EIPS/eip-4824), if added while creating the DAO. "
   daoURI: String
+
+  " Additional information related to the created DAO, if added while creating the DAO. "
   metadata: String
+
+  " Timestamp for the block where the DAO was created. "
   createdAt: BigInt!
+
+  " Tokens held by the DAO. "
   token: ERC20Contract
+
```

Comment:
@Michael-A-Heuer for visibility
this attribute is not accurate any more, but also need some planning

Reply:
Why not?